src/debug_clang.cc

```diff
@@ -463,6 +463,7 @@ void Debug::Clang::remove_breakpoint(const boost::filesystem::path &file_path, i
             auto file_spec=line_entry.GetFileSpec();
             boost::filesystem::path breakpoint_path=file_spec.GetDirectory();
             breakpoint_path/=file_spec.GetFilename();
+            breakpoint_path = boost::filesystem::canonical(breakpoint_path);
```

Comment:
See for instance https://github.com/cppit/jucipp/blob/master/src/juci.cc#L29. Add `error_code ec` and check if `!ec`.